arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -934,7 +934,7 @@
           },
           "description": "Gets or sets the metadata attached to the resource."
         },
-        "TTL": {
+        "ttl": {
```

Comment:
Kirthi is correct. The above change will still be a no-op on the generated code.